spyder/widgets/reporterror.py

```diff
@@ -265,70 +356,57 @@ def set_require_minimum_length(self, state):
 
     def set_github_repo_org(self, repo_fullname):
         """Set the report Github organization and repository."""
-        repo, org = repo_fullname.split('/')
+        org, repo = repo_fullname.split('/')
         self._github_org = org
         self._github_repo = repo
 
     def _submit_to_github(self):
         """Action to take when pressing the submit button."""
-        # Get reference to the main window
-        if self.parent() is not None:
-            if getattr(self.parent(), 'main', False):
-                # This covers the case when the dialog is attached
-                # to the internal console
-                main = self.parent().main
-            else:
-                # Else the dialog is attached to the main window
-                # directly
-                main = self.parent()
-        else:
-            main = None
-
         # Getting description and traceback
         title = self.title.text()
         description = self.input_description.toPlainText()
         traceback = self.error_traceback[:-1]  # Remove last EOL
 
         # Render issue
-        if main is not None:
-            issue_text = main.render_issue(description=description,
+        if self.is_report:
```

Comment:
`self.is_report` denotes the fact that someone is trying to report an issue directly from Spyder (i.e. not something that came from an internal error). Therefore, I don't understand why below you're trying to render the issue with an associated traceback.

Reply:
Fixed